docs/team/limzhenwy.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Lim Zhen Wy's Project Portfolio Page
+---
+
+### Project: FAPro
+
+FApro seeks to improve the quality of life of financial advisors (FAs). It allows FAs to keep track of large numbers of contacts. It allows FAs to have a one-stop platform to manage their contacts and conduct financial analytics while providing a big-picture view of their clientele as a whole.
+
+Given below are my contributions to the project.
+
+* **New Feature**:
+  * What it does:
+  * Justification:
+  * Highlights:
+  * Credits: 
+
+* **New Feature**:
+
+* **Code contributed**:
```

Comment:
Maybe can add the RepoSense link as well? I referred to how Glenn did it.
here's yours!
https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=limzhenwy&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22